docs/performance-tests.md

```diff
@@ -0,0 +1,218 @@
+  through the EVM. This transaction type represents an intermediate level of
+  complexity and provides insights into how efficiently the Mezo Node handles
+  precompiled contracts.
+
+- ERC20 contract transfers: Finally we executed transfers using an actual
+  deployed ERC20 token contract. Unlike the previous categories, these transactions
+  involve full smart contract execution within the EVM. When a transfer is called on
+  this contract, the node must load the contract's code from storage, execute the
+  EVM operations defined in the contract (including balance checks, state updates,
+  and event emissions), and persist the updated state. This transaction type
+  represents realistic workloads commonly seen in production environments and
+  provides the most relevant metrics for evaluating the node's performance for
+  typical DApp operations.
+
+### Metrics observed
+
+The following metrics have been observed while running the perfomances tests:
+
+- Gas consumed per block: This measurement reveals how efficiently the block space
+  is being used and whether transactions are optimally packed into blocks. Consistent
+  gaps between max gas and consumed gas might indicate opportunities to decrease max
+  gas limits, while blocks consistently reaching the maximum suggest potential c
+  ongestion issues that might require parameter adjustments.
+
+- Mempool unconfirmed transaction: The count of pending transactions awaiting to be
+  added in blocks indicates of network congestion. A growing mempool backlog
+  signals that the network cannot process transactions at the rate they're being
+  submitted. For Mezo Node specifically, monitoring this metric is crucial due to
+  its transaction pruning behavior, as excessive mempool growth triggers more
+  frequent re-checking operations that can impact node performance. Persistent
+  mempool growth may indicate the need for increased block capacity or fee market
+  adjustments.
+
+- Transactions per blocks: An easier way to understand the gas consumed per block.
+
+- Time take per block: How long validators takes to construct a valid block,
+  including transaction selection, execution, and state updates. Extended block
+  building times can lead to inconsistent block production intervals and
+  reduced overall throughput (Mezo is trying to achieve block production every
+  3.5 seconds in average). This measurement is especially relevant when
+  evaluating how the re-checking behavior of the mempool affects validator
+  performance under load.
+
+- Gas price: This represent the cost per unit of gas, and reflect the current
+  congestion level of the network. Under load the gas price will increase in an
+  exponential manner, and decrease when the congestion of the network reduce.
+
+### Scripts
+
+The scripts used to run these tests are available in the mezod repository
+[here](https://github.com/mezo-org/mezod/tree/main/tests/perfs).
```

Comment:
Broken link ser

Reply:
correct fixed in: a59aa802860012975aa8494de3f3db12abc426a1, (it's still gonna be broken 'til merge though)